setup.py

```diff
@@ -8,7 +8,7 @@
     long_description = f.read()
 
 # Parse the version from the mapbox module.
-with open('mapbox/__init__.py') as f:
+with open('mapbox/about.py') as f:
```

Comment:
What's the impetus for this change? I'd prefer to keep the version in `__init__` for consistency with other projects.

If we must go this route, we need to remove the `__version__` from `__init__.py` to avoid confusion, instead importing it from `about.py` and exposing it from the package root.